src/lib/layers/lane_details/IntersectionPolygons.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import Layer from "./Layer.svelte";
+  import { caseHelper } from "../../../maplibre_helpers";
+  import type { GeoJSON } from "geojson";
+
+  export let gj: GeoJSON;
+
+  let style = {
+    type: "fill",
+    filter: ["==", ["get", "type"], "intersection"],
+    paint: {
+      "fill-color": caseHelper(
+        "intersection_kind",
+        {
+          MapEdge: "#696",
```

Comment:
The feature properties for all of the layers are not well-documented anywhere right now. I know them from working with osm2streets, but if I can define / clarify anything, let me know. The things here are from https://github.com/a-b-street/osm2streets/blob/0d6f0f973b3fa8dbdc8223aa6f412a8d997252fc/osm2streets/src/intersection.rs#L48